labgrid/driver/quartushpsdriver.py

```diff
@@ -0,0 +1,83 @@
+# pylint: disable=no-member
+import attr
+import subprocess
+import os.path
+import re
+
+from ..factory import target_factory
+from ..protocol import BootstrapProtocol
+from ..resource.remote import NetworkAlteraUSBBlaster
+from ..resource.udev import AlteraUSBBlaster
+from ..step import step
+from .common import Driver, check_file
+from .exception import ExecutionError
+
+
+@target_factory.reg_driver
+@attr.s(cmp=False)
+class QuartusHPSDriver(Driver):
```

Comment:
This needs to implement a Protocol. BootstrapProtocol would be the obvious choice, however your driver implements an additional feature: choosing the address of where to load the file. I am not sure that we want to extend the existing protocol or define a new one for this.